AvaloniaUI/Avalonia
# Api validation tool

how to test/use:
1. Make a breaking change
2. Push
3. See failed Nuke task on the CI
4. Run `nuke` locally (or `nuke --update-api-suppression true` which is true by default on a local machine).
5. Commit updated suppression files with breaking changes defined in them
6. Now nuke task is passing
